.github/actions/minty/main.js

```diff
@@ -0,0 +1,49 @@
+const core = require('@actions/core');
```

Comment:
We should add license, all over

Reply:
Yes, that's part of the final cleanup. Do you feel that this implementation is correct? Mostly just looking for early input on if this is directionally the "right" way to do this.

Reply:
Overall process LGTM.

Reply:
👍